opal/mca/threads/pthreads/threads_pthreads_wait_sync.c

```diff
@@ -6,49 +6,53 @@
  * Copyright (c) 2016      Los Alamos National Security, LLC. All rights
```

Comment:
None of the modifications in this file are justified.

Reply:
`#include "opal/mca/threads/wait_sync.h"` (and according copyright change) is the only change that is truly necessary. The other changes are for unifying the code format in `mca/threads`.

If code-format changes should not be included, I will remove them from this PR.